core/src/main/clojure/xtdb/types.clj

```diff
@@ -1112,13 +1313,19 @@
    [:timestamp-tz :micro] :timestamptz
    :tstz-range :tstz-range
    [:interval :month-day-nano] :interval
+   [:list :i32] :_int4
+   [:list :i64] :_int8
 
-   #_#_ ; FIXME not supported by pgjdbc until we sort #3683 and #3212
-   :transit :transit})
+   #_#_; FIXME not supported by pgjdbc until we sort #3683 and #3212
```

Comment:
nit: try to minimise your diff, where possible, to make it easier for the PR reader to figure out what's changed :pray:

Reply:
do we have a common code formatter / editorconfig?
that might help reduce irrelevant changes. 
but point taken.